lib/commons/dom/find-up.js

```diff
@@ -9,33 +9,24 @@
  * @return {HTMLElement|null} Either the matching HTMLElement or `null` if there was no match
  */
 dom.findUp = function (element, target) {
-	'use strict';
```

Comment:
ok, so the difference here is that if the starting node is the highest node in the shadow DOM, then the old code did not find a matching host node - whereas this code does?